web/store/module.store.ts

```diff
@@ -377,6 +377,7 @@ export class ModulesStore implements IModuleStore {
       runInAction(() => {
         set(this.moduleMap, [moduleId, "link_module"], [response]);
       });
+      this.fetchModuleDetails(workspaceSlug, projectId, moduleId);
```

Comment:
Similar to delete. Just put the details from response in moduleMap.

Reply:
Done